TicTacToe/Program.cs

```diff
@@ -9,238 +10,292 @@ namespace TicTacToe
 {
     class Program
     {
+        const int startPosition = 2;
+        const int spaceBetween = 4;
+        const int numOfBoards = 10;
+        const int innerCells = 3;
+        const int setLocation = 38;
+        const int cellsNumber = innerCells * innerCells;
+        const int distanceBoards = innerCells * spaceBetween;
+        const char player1 = 'X';
+        const char player2 = 'O';
+        const char empty = ' ';
+        static Random rand = new Random();
         static void Main(string[] args)
         {
-            char[][] board = new char[10][];
-            char player1 = 'X';
-            char player2 = 'O';
-            char currentPlayer = 'X';
-            int z = 9;
+            char[][] board = new char[numOfBoards][];
+            char currentPlayer = player2;
+            int z;
 
-            buildBoard();
-            initBoard(board);
+            HomePage();
+            InitBoard(board);
+            BuildBoard(board);
 
-            do
+            while (!SubGameOver(cellsNumber, board, currentPlayer)) 
             {
+                currentPlayer = ChangePlayer(currentPlayer);
                 if (currentPlayer == player1)
```

Comment:
you can use the ?: operator to make the code less verbose: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator

"verbose" is a term referring to how much text there's in the code